File: backend/account_service/internal/dto/task_dto.go
package dto

import (
	"github.com/RustamSafiulin/mesh_cloud_computation/backend/account_service/internal/model"
)

type TaskDto struct {
	ID 		    string `json:"id,omitempty"`
	AccountID   string `json:"account_id,omitempty"`
	Description string `json:"description,omitempty"`
	StartedAt   int64  `json:"started_at"`
	CompletedAt int64  `json:"completed_at"`
	State       int    `json:"state"`
	StateText   string `json:"state_text"`
}

func TaskDtoFromTask(task *model.Task) *TaskDto {

	taskDto := &TaskDto{
		ID:          task.ID.Hex(),
		AccountID:   task.AccountID.Hex(),
		Description: task.Description,
		StartedAt:   task.StartedAt,
		CompletedAt: task.CompletedAt,
		State:       task.State,
		StateText:   model.GetStateStringFromState(task.State),
	}

	return taskDto
}

func TaskDtoListFromTaskList(tasks []model.Task) []TaskDto {

	var result []TaskDto
	for _, task := range tasks {
		taskDto := TaskDtoFromTask(&task)
		result = append(result, *taskDto)
	}

	return result
}

File: desktop_app/MeshDesktopClient/MeshDesktopClientModel/Dto/TaskDto.cs

using System;

using Newtonsoft.Json;

namespace MeshDesktopClient.Dto
{
    public class TaskDto
    {
        #region Props and Fields

        [JsonProperty(PropertyName = "email")]
        public String Id { get; set; }

        [JsonProperty(PropertyName = "account_id")]
        public String AccountId { get; set; }

        [JsonProperty(PropertyName = "description")]
        public String Description { get; set; }

        [JsonProperty(PropertyName = "started_at")]
        public Int64 StartedAt;

        [JsonProperty(PropertyName = "completed_at")]
        public Int64 CompletedAt;

        [JsonProperty(PropertyName = "state")]
        public Int32 State;

        [JsonProperty(PropertyName = "state_text")]
        public String StateText;

        #endregion

        public TaskDto()
        { }
    }
}